src/app/content/studyGuides/components/UsingThisGuide/UsingThisGuideBanner.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled, { css } from 'styled-components/macro';
+import theme from '../../../../theme';
+import desktopBanner from './assets/banner.png';
+import mobileBanner from './assets/banner_mobile.png';
+import { filters } from '../../../styles/PopupConstants';
+import { FormattedMessage } from 'react-intl';
+import { H2, h4MobileStyle } from '../../../../components/Typography/headings';
+import { Times } from 'styled-icons/fa-solid/Times';
+
+const bannerStyles = css`
+  width: 100%;
+  margin: auto;
+`;
```

Comment:
It's only used it one place now so you can probably move it there.
Also i don't think `margin:auto` changes anything